dev-tool.sh

```
@@ -0,0 +1,124 @@
+  fi
+  mvn -pl "$ACTIVE_MODULE" exec:java -Dexec.args="$*"
+}
+
+# Autocompletion function for 'choose'
+_choose_completion() {
+  local cur opts
+  COMPREPLY=()
+  cur="${COMP_WORDS[COMP_CWORD]}"
+  opts=$(list)
+
+  if [ ${COMP_CWORD} -eq 1 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)